packages/webdriverio/src/middlewares.js

```diff
@@ -45,8 +45,33 @@ export const elementErrorHandler = (fn) => (commandName, commandFn) => {
             }).apply(this)
         }
 
-        return fn(commandName, commandFn).apply(this, args)
+        try {
+            return fn(commandName, commandFn).apply(this, args)
```

Comment:
I think this command returns a promise and wouldn't throw on a stale element error

Reply:
CURRENT (not this code) behavior is that the request retries will throw a warning twice, and then rejects with the exception. I'll try a .catch() and see if it works.

Reply:
Ok - after extensive testing, this needs to be a try/catch, as it does NOT return a promise.

Reply:
what does it return?